src/combine.cpp

```diff
@@ -1788,6 +1788,16 @@ void do_symbol_check(chunk_t *prev, chunk_t *pc, chunk_t *next)
             //    using AbstractLinkPtrPtr = AbstractLink**;
             set_chunk_type(pc, CT_PTR_TYPE);
          }
+         else if (  (  get_chunk_parent_type(pc) == CT_FUNC_DEF
+                    && (chunk_is_opening_brace(next) || chunk_is_star(pc->next)))
```

Comment:
Please write the issue number as comment at the begin of your work, line 1791.
Look at some examples, such at line 522:
// Issue # 2166